src/main/java/appeng/recipes/AEItemResolver.java

```diff
@@ -91,14 +91,24 @@ public Object resolveItemByName( final String nameSpace, final String itemName )
 				return this.cableItem( parts.cableSmart(), itemName.substring( itemName.indexOf( '.' ) + 1 ) );
 			}
 
+			if( itemName.equals( "cable_dense_covered" ) )
+			{
+				return new ResolverResultSet( "cable_covered", parts.cableDenseCovered().allStacks( 1 ) );
+			}
+
+			if( itemName.startsWith( "cable_dense_covered." ) )
+			{
+				return this.cableItem( parts.cableDenseCovered(), itemName.substring( itemName.indexOf( '.' ) + 1 ) );
+			}
+
 			if( itemName.equals( "cable_dense_smart" ) )
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move the "cable_dense_smart" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1132)